app/src/main/java/org/piramalswasthya/sakhi/repositories/AbhaIdRepo.kt

```
@@ -827,4 +832,96 @@ class AbhaIdRepo @Inject constructor(
+                        return@withContext NetworkResult.Error(-6, "No match for this face, Please proceed with registered person")
+                    }
+
+                    if (message.contains("reached the maximum limit")){
+                        return@withContext NetworkResult.Error(1227, "ABHA Creation Limit Reached! Please try again late.")
+                    }
+
+                    sendErrorResponse(response)
+                }
+            } catch (e: IOException) {
+                NetworkResult.Error(-1, "Unable to connect to Internet!")
+            } catch (e: JSONException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)